guides/source/routing.md

```diff
@@ -1112,10 +1112,18 @@ For example, here's a small section of the `rake routes` output for a RESTful ro
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
-You may restrict the listing to the routes that map to a particular controller setting the `CONTROLLER` environment variable:
-
+You may restrict the listing to the routes that map to a particular controller using `-c` option or setting the `CONTROLLER` environment variable:
 ```bash
 $ CONTROLLER=users bin/rake routes
+$ bin/rake routes -c users
+$ bin/rake routes -c users_controller
```

Comment:
Does the `CONTROLLER` env var support the `_controller` suffix? If no, I don't think we have to. Besides the option is short for "search for a controller" so I think it's to just support "-c users" and infer that as "-c users_controller".